helm/vertical-pod-autoscaler-app/values.yaml

```diff
@@ -48,7 +48,7 @@ recommender:
     # recommender.image.pullPolicy -- The pull policy for the recommender image. Recommend not changing this
     pullPolicy: IfNotPresent
     # recommender.image.tag -- Overrides the image tag whose default is the chart appVersion
-    tag: ""
+    tag: "0.10.0-oomfix"
```

Comment:
can't set this in Chart.yaml as it would also use the same tag for the admission controller

Reply:
works for me. But I would add a comment directly pointing to the issue.